--- a/odoorpc/tools.py
+++ b/odoorpc/tools.py
@@ -3,6 +3,7 @@
 #
 #    OdooRPC
 #    Copyright (C) 2014 Sébastien Alix.
+#    Copyright 2021 Vincent Hatakeyama
 #
 #    This program is free software: you can redistribute it and/or modify
 #    it under the terms of the GNU Lesser General Public License as published
@@ -22,8 +23,13 @@
 manage the configuration related to an instance of :class:`ODOO <odoorpc.ODOO>`,
 and some useful helper functions used internally in `OdooRPC`.
 """
-import collections
 import re
+import sys
+
+if sys.version_info >= (3, 3):
+    from collections.abc import MutableMapping
+else:
+    from collections import MutableMapping
 
 MATCH_VERSION = re.compile(r'[^\d.]')
 
@@ -30,3 +34,3 @@
 
-class Config(collections.MutableMapping):
+class Config(MutableMapping):
     """Class which manage the configuration of an
     :class:`ODOO <odoorpc.ODOO>` instance.